packages/node_modules/@ciscospark/react-container-file-uploader/src/selectors.js

```diff
@@ -0,0 +1,18 @@
+import {createSelector} from 'reselect';
+
+const getSpark = (state, ownProps) => ownProps.sparkInstance || state.spark.get('spark');
+const getConversation = (state) => state.conversation;
+const getActivity = (state) => state.activity;
+
+const getContainerProps = createSelector(
```

Comment:
Should we create a selector test for this? Or let the existing tests cover this path?

Reply:
There really isn't anything to test on this, but we could. It really just helps memoize the parts of the store that we want for the container.